pytgcalls/pytgcalls/mtproto/telethon_bridge.py

```diff
@@ -162,10 +167,7 @@ def register_update_handlers(self):
         self.client.add_event_handler(self._process_update, Raw)
 
     async def resolve_and_set_join_as(self, join_as):
-        my_peer = await self.get_and_set_self_peer()
```

Comment:
`join_as`can be none... by default it should be self peer. look into pyrogram impl

Reply:
[**@MarshalX**](https://github.com/MarshalX) commented on [Jul 17, 2021, 12:22 PM GMT+3](https://github.com/MarshalX/tgcalls/pull/102#discussion_r671662533 "2021-07-17T09:22:03Z - Replied by Github Reply Comments"):
> `join_as`can be none... by default it should be self peer. look into pyrogram impl

Yes. Ik. I was editing it after I made my pr :) Should be updated now

Reply:
Wait